lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -92,13 +94,16 @@ func GetHandlers() []obsidian.Handler {
 
 const (
 	mobilitydStateExpectedMatchCount = 2
+	imsiExpectedMatchCount           = 2
 )
 
 var (
 	// mobilitydStateKeyRe captures the IMSI portion of mobilityd state keys.
 	// Mobilityd states are keyed as <IMSI>.<APN>.
 	mobilitydStateKeyRe = regexp.MustCompile(`^(?P<imsi>IMSI\d+)\..+$`)
 
+	// imsiKeyRe captures the IMSI portion of any IMSI-prefixed ID strings
+	imsiKeyRe                    = regexp.MustCompile(`^(?P<imsi>IMSI\d+)\.?.*$`)
```

Comment:
As described below: you'll probably end up wanting to just use `mobilitydStateKeyRe` and can remove these new vars